webpack.test.config.js

```diff
@@ -3,8 +3,9 @@
 var path          = require('path');
 var _             = require('underscore');
 var commonConfig  = require('./webpack.common.config');
-var testConfig    = commonConfig('main-tests.js');
 var plugins       = require('./buildtools/webpack/plugins');
+var testConfig    = commonConfig('main-tests.js');
```

Comment:
Is this just a cosmetic change?

Reply:
More of an `eslint` change. All of our imports are now at the top of the page (`commonConfig` slipped in there before).